### DISCUSSION TITLE
Updates? 😐

### DISCUSSION THREAD

**Discussion by GraphicOwls** [notaapp/talk#402] [Questions]
Hello,

Just curious what happened to the updates? 2 months before the previous update, you guys mentioned that updates were going to be much more frequent because you rewrote a lot of the base code, which is going to make progression on the app quicker and more efficient. However, sadly there has been nothing since. Just curious if the app is still actively being worked on.

Thanks,
– Noah

**Comment by marlonjames71** [comment]
Was going to ask the same thing.

**Comment by astoilkov** [comment]
Hey, sorry you feel bad about the frequency of updates. I do too.

We're spending some time learning and experimenting with LLMs. We feel like this tech will have an important role in note-taking, and knowledge management, and we feel it's important that we understand it. We already have some ideas on how to use it for note-taking (other than what is mainstream). While we really want to create new updates constantly and excite people around the product, we also want to maximize our value creation and built something really useful for the users and ourselves — that's always a hard balance.

We're also working on the next update of Nota. We've already implemented some of the things we've planned for the release.

**Reply by GraphicOwls** [reply]
I completely understand what you're saying about LLM's and getting into that space as well as building something useful for the users. 

At the same time though, people who support Nota like myself (paid users) have requested updates or changes within it's current state, and then it seems like they may have been forgotten in some cases. So, it's not the "frequency of updates," but rather whether the community is being heard.

I hope you understand where I'm coming from, and I only mean well. I just feel like a lot of writing apps tend start moving onto other new things without addressing core related issues first these days. Especially ones frequently mentioned by your dedicated user base, but I am excited for the next update for Nota.

Thanks!

**Reply by claudiorrrr** [reply]
I support this comment above.

I also have a Nota license because at least to me Nota is probably the best note-taking app all around. Please don't let the app die.

**Reply by astoilkov** [reply]
Hey, thanks for sharing.

I see an opportunity for us to improve here. Are you willing (both of you) to make a quick 15-20 min call? If yes, write to me at hello@astoilkov.com. We can discuss what features you want and explore why you feel this way.

**Reply by lolbat** [reply]
I don't really understand this rush to stuff LLM features into every app. And given that almost every app already has plans to put LLM features into their app I don't really see what advantage you think you will get.

**Reply by astoilkov** [reply]
It's more like "what's the disadvantage of not including it". AI is a powerful tool that might be considered essential in the near future. However, we're not sure, this is why we need to explore and learn more so we can make better decisions.

Also, we think we have ideas that we still haven't seen elsewhere.

**Reply by tylxm** [reply]
@astoilkov Can you share what kind of ideas? like generating markdown table automatically using AI?

**Reply by astoilkov** [reply]
One side is generation of content, that's the thing I see the most from others. What we're exploring is connections between notes and how to make sense of the things you've written.

We're still experimenting, I'm still not sure where that would lead us.

**Reply by GraphicOwls** [reply]
I hope I'm not out of place here. Earlier you mentioned that Nota has been quiet because you guys were researching LLM's and how to integrate it into Nota. However, that's not why you have the users you do. None of this existed when I purchased Nota, and from most of these other comments it seems like they didn't either. 

I agree with @lolbat, why the rush? I know that I love writing in Nota. I get this awesome VS Code writing experience and other nice things that _no other_ app has. This is what drew me to purchasing a subscription. I wish the focus was more aligned with what made you guys create Nota in the first place (before all this LLM stuff), as well as what us users are requesting.

I could care less for having ai connect notes for me. I can find my notes perfectly fine myself. I also have Chat GPT plus, and if I _really_ need to I can get ideas from there and bring them into Nota. _If I need to_. Personally, I would just love to see the focus go back into Nota organically instead of being sucked into this ai world. I mean have you really had that many Nota users express an absolute necessity for ai in the app? With the way the app is, I just can't see that. I could be wrong, but it's not something I would bet on. 

If I am being honest, I would assume that as devs working on the app, maybe some boredom kicked in after working on it for as long as you have, and you wanted to jump on something new for the brain challenge, which I totally get. To feel excited again, or maybe it was pure curiosity, I don't know. Maybe I am just wrong altogether lol.

**Reply by tylxm** [reply]
I'm not a big fan of LLM in note taking app either.

**Reply by lolbat** [reply]
> What we're exploring is connections between notes and how to make sense of the things you've written.

Isn't that my job? 

I want UI fixes. I don't want LLM features. I can think for myself.

**Reply by minthemiddle** [reply]
> What we're exploring is connections between notes and how to make sense of the things you've written.

What I am dreaming of is local semantic search within my notes. Notes creation is easy, notes application is difficult, especially with older notes. Tagging, linking, all of that is of course possible but does not trump semantic relateness. 

I could build a system with LlamaIndex or LangChain (basically: creating local embeddings and GPT API calls) but that is very complicated and error-prone and hard to maintain.

Using LLMs to get semantic access to my thoughts would enhance my note taking and management a lot.

**Reply by sguergachi** [reply]
It's funny because the iA guys spent a year doing the same exploration and determined they wouldn't integrate any LLM tech into their app, instead the best thing to do was to help track the AI content pasted in vs. the content you wrote yourself. https://ia.net/topics/writing-with-ai

**Comment by tylxm** [comment]
Just purchased the license, hope you guys continue to support this product.

**Comment by borekb** [comment]
I'll just quickly pop in here to say that the guys behind Nota, @erusev and @astoilkov, have been extremely reliable in the past when it comes to good stewardship of this project, and [Caret editor](https://caret.io/) before that.

I've been following their work for many years and yes, sometimes the updates take some time, but the result is always well thought-out and brings joy when using the product(s).

I have no affiliation with them but just came here to say that sometimes, it's worth waiting for great things. I'm sure you all guys enjoy Nota and it's also because they take time to build things the right way, not always the fast way. Nota creators are great guys!

**Reply by astoilkov** [reply]
Hey Borek. Once again, thank you for your words and support throughout the years. I still remember your [Caret rocks!](https://github.com/careteditor/issues/issues/122) issue. One of those small things that make a difference for others.

Also, shout out to @minthemiddle, who created the same kind of issue for Nota  — [Nota is great!](https://github.com/notaapp/talk/discussions/341).

They both end with exclamation marks! 😄

**Reply by minthemiddle** [reply]
… and I still use and love Nota's simplicity and limited set of features every single day. For me, Nota is like Things3 but for notes.

**Comment by webwesen** [comment]
regular monthly releases fixing even some of the most minor bugs would go a long way supporting the community you have built around Nota. (paid customer here)

**Reply by astoilkov** [reply]
Yeah, I agree. We can do that.

Are you referring to specific bugs? If you can link them I can understand better what type of bugs we should prioritize. I would assume that the issue here is more about not recognizing the bug is important enough for us to spend the time going through the whole release process.

**Reply by webwesen** [reply]
nothing in particular... anything. just optics - showing that the product is still alive and is no longer an abandonware.

**Reply by astoilkov** [reply]
Got it. Yeah, you mean that more frequent updates (regardless of the size) just feel better for the users. I like that. It actually feels better for us too.

> and is no longer an abandonware.

What do you mean by "no longer" (we've been doing Nota releases for 2 years and a half)?

**Reply by GraphicOwls** [reply]
Totally agree with this. I didn't think Nota was even being worked on anymore.

**Reply by minthemiddle** [reply]
Paying customer here and I am totally fine with thorough less frequent releases.

**Comment by marlonjames71** [comment]
Was told UI updates were coming a while ago. No big deal if it's not feasible at the moment. I just like when words are kept.

**Reply by erusev** [reply]
We are working on a UI update, and as Nota users ourselves, we are excited about it, but it's hard to say when it will be ready. Sorry for the wait 😔

**Reply by marlonjames71** [reply]
Excited to see it. Also, are you guys still looking into a phone app to access notes on the go? That's a big pain point for me right now.

**Reply by erusev** [reply]
We have a mobile app in the works, but it's mostly focused on reading and searching notes, less on editing. It's on pause for now, but we hope to get back to it at some point.

**Reply by marlonjames71** [reply]
Ehh...you guys kinda lost me. Kind of a shame to be honest.

**Comment by astoilkov** [comment]
There has been a lot of activity in this thread. We agree that we should be more informative, especially when releases are too far apart. 

I had a call with @claudiorrrr yesterday and he proposed the idea of a place where we can discuss the new updates. I just created a new discussion that discusses the new upcoming release — https://github.com/notaapp/talk/discussions/405.

Thanks for all the feedback and help. We're very fortunate to have such an amazing user base.

**Reply by webwesen** [reply]
checking in... 1 month after ^^^ this message. any updates? thanks!

**Reply by marlonjames71** [reply]
I think Nota is dead. Looks like they're just focusing on Intellibar and chatgpt

**Reply by igtsimoes** [reply]
2 months later... 😞

**Reply by GraphicOwls** [reply]
Seriously.. waste of money! Their intentions should've been more clear or I never would have purchased the app. I can't stand that!

**Reply by minthemiddle** [reply]
Yes, more transparency would be appreciated. And: I have been getting value out of Nota from the very first moment and still do to the very moment. I don't need new features, I would appreciate stability and speed updates though.

**Reply by GraphicOwls** [reply]
@minthemiddle How do you get value out of Nota? Can you not write notes in any of the other 100+ note apps out there? I paid to use Nota of course, but also to support the growth and future of the app. And it's apparent now that there isn't one. That would be misleading.

**Reply by minthemiddle** [reply]
> How do you get value out of Nota? Can you not write notes in any of the other 100+ note apps out there? 

It stays out of my way, is fast, is simple. I don't know any other app that feels likes this (and yes, I know Obsidian et al.). I use it every day, so I don't depend on future changes. I understand that it's very different for you.

**Reply by webwesen** [reply]
@minthemiddle - continuous support and development is an absolute requirement as manifested in https://github.com/notaapp/talk/issues/410

**Comment by igor-arkhipov** [comment]
I don't think it's worth forcing release of new versions. Quality is better than frequency.

Somehow make regular "releases" to calm down users and it looks funny
![](https://github.com/notaapp/talk/assets/18555898/a33ea657-0f50-43f6-8478-2991fb64d37a)

**Comment by webwesen** [comment]
5 months have passed... a keep-alive ping: "any updates?"

**Comment by igtsimoes** [comment]
Another ping... just in case... are we getting a Christmas update gift?

**Comment by arunaugustine** [comment]
Just want to chime in and say, I love Nota as it is today! Otherwise I would never go looking to see what's brewing. 
If updates are not needed, or will take time to brew correctly, that's OK! 

Just wanted to check in and see if you guys were doing OK! @astoilkov and @erusev 

I just hope that you guys don't think other apps like Obsidian has won this space due to their feature list (or bloat if you ask me) and give up on Nota.

**Comment by erusev** [comment]
Thanks for the kind words, @arunaugustine. We're doing well, thanks for asking ❤️

Earlier this year, we took on a new project, [IntelliBar](https://intellibar.app/), and development on Nota slowed down a bit, but we are working on it and using it daily.

The time away from Nota has given us a clearer view of how to improve it, and we're excited to make it happen. We'll have more to share in the new year.

**Reply by igor-arkhipov** [reply]
Apple just killed all ChatGPT-wrapper-apps on WWDC2024

**Comment by denvist** [comment]
After more than a year since the last release, it looks like the project is going to be abandoned. I think it is time to look for alternatives.

**Reply by nelsonfigueroa** [reply]
I switched to [Obsidian](https://obsidian.md/) recently. I think it's the best alternative although it's more bloated than Nota and the UI isn't as clean (in my opinion). Obsidian does get regular updates though. I hope I can go back to Nota some day.

**Reply by denvist** [reply]
@nelsonfigueroa, yes, Obsidian is great, but it's not free for commercial use.

**Comment by HwiNyeonKim** [comment]
Still waiting an update for Table on 0.41

**Comment by marlonjames71** [comment]
It's a shame Nota has been abandoned. It was on its way to being the best notes app. Obsidian is too distracting, others don't have the same writing/editing experience or are fractured in many ways. Others also don't provide the same sense of ownership over your data. Nota had a great balance. Such a shame.

**Reply by minthemiddle** [reply]
I went back from Obsidian to Nota because of its simplicity. It feels like the Things equivalent of writing. I'm actually pretty happy with the set of features it has, yes, I would love to see some extras, but being limited to status quo surprisingly keeps my focus on managing notes not tweaking or playing with the app. (Still, some feedback from the devs to at least keep the app updated or give us an idea about what options are there for the future would be highly appreciated).

**Comment by erusev** [comment]
We spent more than 5 years working on Caret and then Nota. We have by far the most incentive to keep the app alive and are looking for ways to make that happen.

**Reply by marlonjames71** [reply]
What do you mean looking for ways to keep it alive? Like financially?

**Reply by erusev** [reply]
Yes, financially is a significant part of it.

**Reply by igor-arkhipov** [reply]
Сonsider starting a patreon / ko-fi / etc. E.g. I bought Mailmate.app years ago (really best in class) and I sponsor developer via patreon, because he deliver updates [quite often](https://updates.mailmate-app.com/release_notes_archive/MailMate_r6039.html)

**Reply by marlonjames71** [reply]
> Yes, financially is a significant part of it.

Bear has a very reasonably priced subscription. Have you thought about that? Might need to earn folks trust back. But, once you do, I think folks would be happy to pay it.

**Reply by webwesen** [reply]
> Сonsider starting a patreon / ko-fi / etc. E.g. I bought Mailmate.app years ago (really best in class) and I sponsor developer via patreon, because he deliver updates [quite often](https://updates.mailmate-app.com/release_notes_archive/MailMate_r6039.html)

was exactly my point a year ago - small & frequent keep-alive releases (like Mailmate) build trust and loyal following. disappearing for a year does the opposite.

**Comment by denvist** [comment]
Hooray! 0.41.0 was released!

**Reply by minthemiddle** [reply]
Thank you for the new release, @astoilkov  and @erusev !

**Comment by ace-dent** [comment]
Congratulations @erusev on a nice update, especially the enhanced table functionality! 🙏🥳

**Reply by mathewmitchell** [reply]
I've updated, but I can't tell what changes were made. Any documentation somewhere about this?

**Reply by nelsonfigueroa** [reply]
Check out https://nota.md/changelog

**Reply by mathewmitchell** [reply]
Many thanks! That does it. 

Unless I'm missing something there's no direct link to this changelog page within Nota. I would expect it to be under the Help menu. 

... another thing I'm reminded of ... on the official Nota page it says you can "join the waitlist" or "preorder". Both of these options communicate the software is not ready to use, except for the adventuresome. If the developers are confused/disappointed by their sales ... I just want to let you know your Home page isn't helping at all. It actively (but unintentionally) discourages folks from purchasing Nota at this point in time (or earlier). Many many folks know about the misadventures of nvUltra. Too easy to think this product is the same ... vaporware. The developers may be good coders, but are atrocious at business.

**Reply by ace-dent** [reply]
@mathewmitchell - perhaps it's a language issue(?) but your support and encouragement isn't communicated in a very positive way. I believe this very small team has created a beautiful product and we all hope for their success ☺️

**Reply by erusev** [reply]
@mathewmitchell You're right, the site needs an update, and I hope we can get to it soon.

**Comment by marlonjames71** [comment]
Nice little update. I dig the changes to the table. Very easy to manipulate.

**Comment by denvist** [comment]
There is no news, updates, nothing...

**Reply by nelsonfigueroa** [reply]
it is time to move on :)

**Reply by ace-dent** [reply]
> There is no news, updates, nothing...

The App is beautifully crafted and very stable. I decided to purchase a copy this month, to show my support and encourage these indie developers. 🚀

**Reply by denvist** [reply]
@ace-dent 

Like many others here, I have purchased Nota. However, I am concerned that the developers are not communicating with us and seem to have no clear vision for future development.

Let me remind you that the developers previously had a programme called [Caret](https://caret.io). They encouraged people to buy it while it was still in development. But then, after several years, development was discontinued. The main problem is that users are left alone with their [issues](https://github.com/careteditor/issues/issues) since the source code is closed.

Why am I bringing this up? The developers are creating a good product, but it seems that the [same fate](https://github.com/careteditor/issues/issues/911) may befall this project as befell Caret.
